taxonomy-manager-rest-server/src/main/resources/tcp.xml

```diff
@@ -0,0 +1,48 @@
+<!--
+    TCP based stack, with flow control and message bundling. This is usually used when IP
+    multicasting cannot be used in a network, e.g. because it is disabled (routers discard multicast).
+    Note that TCP.bind_addr and TCPPING.initial_hosts should be set, possibly via system properties, e.g.
+    -Djgroups.bind_addr=192.168.5.2 and -Djgroups.tcpping.initial_hosts=192.168.5.2[7800]
+    author: Bela Ban
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns="urn:org:jgroups"
+		xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/jgroups.xsd">
+	<TCP
+			bind_addr="${bind_addr:127.0.0.1}"
+			bind_port="7800"
+			recv_buf_size="${tcp.recv_buf_size:130k}"
+			send_buf_size="${tcp.send_buf_size:130k}"
+			max_bundle_size="64K"
+			sock_conn_timeout="300"
+
+			enable_diagnostics="true"
+			diag_enable_udp="false"
+			diag_enable_tcp="true"
+
+			thread_pool.min_threads="0"
+			thread_pool.max_threads="20"
+			thread_pool.keep_alive_time="30000"/>
+
+	<TCPPING async_discovery="true"
```

Comment:
I might be missing something, but isn't `TCPPING` a static configuration, whereby you need to know in advance the static IP addresses of the members of the cluster in advance? In this configuration, it looks like we're clustering with the local instance only and nothing else?

Reply:
The `${initial_hosts:127.0.0.1[7800],127.0.0.1[7801]}"` bit gets templated with the value of `jgroups.initial_hosts` at runtime, which I was going to create a headless service for (like ELasticsearch).

Reply:
Yeah I see that now. My bad.

Should there be a corresponding PR in `taxonomy-manager-infra` for the new service and environment variable passing to the deployment?

Reply:
Yeah, though it depended on this approach being +1'd first. I'll open it now.